e2e/summaries.py

```diff
@@ -56,7 +51,7 @@ async def run():
     job = await create_summary()
     job_id = job.get('id')
 
-    await sleep_progress(5, 'Waiting for summary to be processed')
+    await sleep_progress(7, 'Waiting for summary to be processed')
```

Comment:
can't we poll the endpoint with a timeout instead?

Reply:
hah, coincidentally i actually changed that to exactly this what you suggested